src/main/java/com/efficio/fieldbook/web/trial/controller/ImportGermplasmListController.java

```diff
@@ -274,11 +284,11 @@ private void saveStudyGermplasm(final int studyId) {
 			final List<ImportedGermplasm> importedGermplasm = importedGermplasmList.getImportedGermplasms();
 			final List<StudyGermplasmDto> studyGermplasmDtoList = this.studyGermplasmTransformer.transformToStudyGermplasmDto(importedGermplasm);
 			// Delete the existing stocks so that we can replace it with the current list.
-			this.studyGermplasmService.deleteStudyGermplasm(studyId);
```

Comment:
As long as we are standardizing, can we rename StudyGermplasmService and StudyGermplasmServiceImpl (and its variable declaration in using classes) to `StudyEntriesService` and `StudyEntriesServiceImpl`